### ISSUE TITLE
Does this package support AHK version 2 yet?

### ISSUE THREAD

**Issue by gtusr** [Perlence/AutoHotkey.py#23]
AHK version 2 was officially released a few weeks ago.

**Comment by Perlence** [comment]
No, currently AutoHotkey.py doesn't support AutoHotkey v2.0. I may consider implementing it, but I don't think it would be practical (or desirable) to support both AutoHotkey v1.1 and v2.0.

**Comment by trajano** [comment]
What the general diff between AHK 2 and AHK 1.1 in terms of the COM API contract ?

**Comment by gtusr** [comment]
I don't know about any changes with respect to the COM API contract.  

My understanding is that version 2.0 was introduced to clean up some of the more baroque parts of the syntax inversion 1, which is now officially in legacy maintenance mode.
